### PR TITLE
GAWB-3947: update serviceTest version to get bugfixes [risk: low]

### PR DESCRIPTION
**Pull request by davidangb**
update serviceTest version in automation, to pick up this bugfix from workbench-libs:

> bugfix: BillingFixtures.claimGPAllocProject now uses the proper OAuth scopes in the case where GPAlloc did not return a project and therefore the calling test needs to create one on the fly.

-----

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by davidangb** [comment]
most tests passed, orch and ui tests each had one failure:

orch
```
[info] *** 1 TEST FAILED ***
[error] Failed: Total 25, Failed 1, Errors 0, Passed 24
[error] Failed tests:
[error] 	org.broadinstitute.dsde.test.api.orch.OrchestrationApiSpec
[error] (test:testOnly) sbt.TestsFailedException: Tests unsuccessful
```

ui
```
[info] *** 1 TEST FAILED ***
[error] Failed: Total 123, Failed 1, Errors 0, Passed 122, Ignored 7
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.metadata.DataSpec
[error] (Test / testOnly) sbt.TestsFailedException: Tests unsuccessful
```

I am re-running these manually.

**Comment by davidangb** [comment]
dockerhub outage caused problems with automated tests. jenkins retest.

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala]: one of the changes between serviceTest 0.10 and 0.13 is that we no longer request `CLOUD_PLATFORM` scope by default when making test-user tokens (b/c we don't request it at runtime for end users). That change broke this test, so I had to update the test.

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala
Comment (dvoet) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala]: is this true? then why do we have this api?

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala
Comment (gpolumbo-broad) [inline automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala:43]: Why do we need the test at all then?  It sounds pretty artificial and therefore I would expect of questionable value.

**Review by dvoet — APPROVED** [review]
1 question but LGTM

**Review by gpolumbo-broad — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala
Comment (Qi77Qi) [reply automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala]: I think you can just do `AuthTokenScopes.userLoginScopes +BigqueryScopes.CLOUD_PLATFORM`?

**Comment by davidangb** [comment]
@gpolumbo-broad @dvoet Can I defer to you on whether or not the add/remove google role API is needed, and whether or not this test is needed? I don't have enough context to really answer that.

I see a couple paths:
1. merge this PR as-is, do nothing else
2. merge this PR as-is, create follow-on stories to decide on removing test & apis
3. delete the test before merging this PR, create follow-on stories to decide on removing apis

any preferences?

**Comment by davidangb** [comment]
following up on this: a Kibana search shows that this API *is* in use - by AoU if nothing else. So it seems the test is a useful and valid test.

**Comment by rushtong** [comment]
@davidangb - Yes, AoU does call this endpoint. We have the need to add a user to (and remove from) a billing project. We call the orchestration endpoint with the role `user` to do that. Relevant aou code:
* https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/fireCloud.yaml#L105
* https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java#L152

**Pull request merged**
